src/main/java/org/sonarlint/intellij/ui/UiUtils.kt

```diff
@@ -27,11 +27,10 @@ class UiUtils {
     companion object {
         @JvmStatic
         fun runOnUiThread(project: Project, runnable: Runnable) {
-            runOnUiThread(project, runnable, ModalityState.defaultModalityState())
+            runOnUiThread(project, ModalityState.defaultModalityState(), runnable)
         }
 
-        @JvmStatic
-        fun runOnUiThread(project: Project, runnable: Runnable, modality: ModalityState) {
+        fun runOnUiThread(project: Project, modality: ModalityState, runnable: Runnable) {
```

Comment:
Just as a question, what's the reason for this change?

Reply:
I prefer to have the Runnable as the last parameter this way I can use Kotlin [trailing lambdas](https://kotlinlang.org/docs/lambdas.html#passing-trailing-lambdas)